Holy_locket.WebAPI/Controllers/DoctorsController.cs

```diff
@@ -0,0 +1,139 @@
+﻿using Holy_locket.WebAPI.TestModels;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.IO;
+using System.Numerics;
+using System.Reflection.PortableExecutable;
+using System.Text.Json;
+
+namespace Holy_locket.WebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    //[Route("Doctors")]
+    public class DoctorsController : ControllerBase
+    {
+        List<Doctor> doctors = new List<Doctor>();
+
+
+        public DoctorsController()
+        {
+
+            string fileName = "Doctors.json";
+
+
+            string jsonString;
+            using (StreamReader reader = new StreamReader(fileName))
+            {
+                jsonString = reader.ReadToEnd();
+            }
+
+
+            doctors = JsonSerializer.Deserialize<List<Doctor>>(jsonString);
```

Comment:
WebAPI shouldn't know how you deal with data, please move any behavior with data to the appropriate project.